### ISSUE TITLE
[ACTIVE_STALE_TAKEOVER_ARBITRATION_WIN] D03|PR475|RICH-CODE-SOURCE-ADMISSION|MIT-WHITESPACE-FALSE-NEGATIVE-REPAIR-V1

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1939]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
SWARM_WORKER_ID: SWARM-PULSE-RICH-20260914
STATUS: ACTIVE_STALE_TAKEOVER_ARBITRATION_WIN
SWARM_LANE_KEY: D03|PR475|RICH-CODE-SOURCE-ADMISSION|MIT-WHITESPACE-FALSE-NEGATIVE-REPAIR-V1
SELECTED_WORK_KIND: SAME_LINEAGE_DEFECT_REPAIR / TERMINAL SOURCE CONVERGENCE
PARENT_ISSUES: #4 #548 #723 #1868
TARGET_PR: #475
TARGET_BRANCH: next100-051/code-rich-source-admission-20260826
TARGET_HEAD_AT_WIN: a3536fba86baba3c3dea959a1548f0fa9ba2f924
LIVE_MAIN_AT_WIN: 9174d65a391dc712d666da8b834b0548e988290f
STALE_PREDECESSOR: #1868
PREDECESSOR_CREATED_UTC: 2026-09-14T05:01:34Z
PREDECESSOR_LAST_UPDATED_UTC: 2026-09-14T05:01:34Z
PREDECESSOR_COMMENTS: 0
LOCAL_FREE_ONLY: true
MERGE_AUTHORITY: false
TRAINING_CLAIM: false

STALE-TAKEOVER / POSTCLAIM ARBITRATION — WIN:
- #1868 never completed its mandatory postclaim arbitration and has zero comments/renewals; its update timestamp is more than six hours before this successor claim;
- mandatory direct OPEN issue collection was enumerated newest-first at 100/page: pages 1..13 populated; page 14 physical exact `[]` sentinel;
- separate direct OPEN PR collection was enumerated newest-first at 100/page: pages 1..6 populated; page 7 physical exact `[]` sentinel;
- targeted exact semantic refresh after both complete sweeps returns only stale predecessor #1868 and this successor #1939;
- final race refresh immediately before Product work: PR #475 remains OPEN/non-draft/mergeable=true on unchanged head a3536fba86baba3c3dea959a1548f0fa9ba2f924; main remains 9174d65a391dc712d666da8b834b0548e988290f;
- current main registry v6 does not contain github:Textualize/rich in credited prior code families or the six-family PR818 terminal bulk intake, so Rich is not superseded and still adds unique potential source-family capacity.
PRODUCT_MUTATION_BEFORE_ARBITRATION: false

KNOWN ROOT CAUSE:
Historical exact-head dedicated Rich verifier failed semantic MIT phrase membership because the immutable 1,056-byte license wraps grant prose across lines. Existing independent PR comments prescribe whitespace normalization only for decoded prose comparison while exact raw size/Git-blob/SHA-256 identity gates stay unchanged.

OWNED SURFACE:
- SAME existing PR #475 / SAME branch only;
- minimal license-semantic normalization repair in tools/qualify_next100_051_rich.py;
- directly necessary focused regression/current-main convergence under canonical shared-CI policy;
- removal of obsolete specialist-workflow surface only if current shared-CI policy and current-main convergence require it;
- truthful exact-head handoff metadata.

AVOID:
- no global registry/dedup/decontam/balance/split/pack/tokenizer/trainer/recovery/evaluation/final-test mutation;
- no source-scope expansion or rights weakening;
- no central integration/merge to main;
- no paid compute, foreign pretrained weights, or external LLM/API corpus intelligence.

ACCEPTANCE:
1. Preserve Rich v15.0.0 exact commit 6ac483cbea39cab124dfd3483bba70ffafb71050, six selected source blobs, license blob 4415505566f261c802b671426be529a31f914137, license SHA-256 deed7c17a4318158190a3ea239cc879a5a50271cebb98ae7025f48fbe58dca15, bounded 46,162 bytes and one-family semantics.
2. Whitespace normalization affects only decoded prose phrase matching; raw license identity gates remain byte-exact and semantic grant drift still fails closed.
3. Converge onto current shared-CI policy without resurrecting obsolete specialist-workflow fan-out.
4. Fresh exact-head current canonical CI path must reach terminal SUCCESS; queued/running/red is not PASS.
5. No canonical capacity promotion, tokenizer fit, optimized-target exposure, optimizer/training, learned weights, final-test use or readiness-percent advance inferred.
6. Release immutable head for fresh different-worker audit/integration; no self-merge.

TRUTH_BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false_FOR_THIS_SOURCE_REPAIR_ONLY
CURRENT_RETAINED_CORPUS_EXTERNAL_LLM_CONTAMINATION=TRUE_VIA_NOMIS1864_PENDING_1874_1878_REPAIR
